lib/recording/recordings.ex

```diff
@@ -0,0 +1,168 @@
+defmodule Onvif.Recording.Recordings do
```

Comment:
```suggestion
defmodule Onvif.Recording do
```
Same as above, without the module stuttering